lib/galaxy/model/store/__init__.py

```diff
@@ -2076,67 +2076,73 @@ def export_jobs(
                         if include_job_data:
                             self.add_dataset(assoc.dataset)
 
-                for assoc in job.output_datasets:
+                for assoc2 in job.output_datasets:
```

Comment:
Not a big difference, but how about renaming these to something like:
- `assoc` -> `id_assoc` (Input dataset)
- `assoc2` -> `od_assoc` (Output dataset)
- `assoc3` -> `idc_assoc` (Input dataset collection)
- `assoc4` -> `idce_assoc` (Input dataset collection element)
- `assoc5` -> `odci_assoc` (Output dataset collection instances)
- `assoc6` -> `odc_assoc` (Output dataset collection)

Feel free to ignore though, the scope of those variables is so small that it is a minimal readability improvement.

Reply:
Oh, this is so much better! I'll make the change. Thanks!